draft-ietf-taps-arch.md

```diff
@@ -329,7 +329,21 @@ The Transport System Implementation Concepts define the set of objects used inte
 
 * Path Selection: Path Selection represents the act of choosing one or more paths that are available to use based on the Path Selection Properties provided by the application, and a Transport Services system's policies and heuristics.
 
-* Protocol Selection: Protocol Selection represents the act of choosing one or more sets of protocol options that are available to use based on the Protocol Properties provided by the application, and a Transport Services system's policies and heuristics.
+* Protocol Selection: Protocol Selection represents the act of choosing one or more sets of protocol options that are available to use based on the Protocol Properties provided by the application, and a Transport Services system's policies and heuristics. See {{security-equivalence}} for considerations regarding security Protocol Selection.
+
+#### Security Protocol Gathering and Equivalence {#security-equivalence}
+
+In some cases, a Transport Services system's policies and heuristics, or individual 
+Protocol Properties, heuristics may yield multiple candidate security protocols. For example, 
+both DTLS and TLS may be viable if an application does not express options that
+are prohibited by either UDP or TCP. In such cases, only one security protocol
+SHOULD be used as the outcome of Protocol Selection as deemed appropriate by 
+System Policy. Multiple protocols MAY be used if they are equivalent in terms 
```

Comment:
I don't understand this should – does that mean we don't want racing between e.g. DTLS and QUIC because they use different security protocols? It is clear that stacking arbitrarily security protocols is a bad idea but this sentence does tell me no racing between security protocols with the next one realativising this.

Reply:
so there's apparent confusion here between "select for attempted racing" and "use in the configured stack". The latter is clearly nonsensical. The former is also potentially problematic, and I think that's what this SHOULD is trying to avoid...

Reply:
Yes, it's pointed at the former. Basically, do NOT race DTLS and QUIC.